rebar.config.script

```diff
@@ -8,4 +9,16 @@ case erlang:system_info(otp_release) =< "R15B01" of
                 CONFIG ++ [{erl_opts, HashDefine}]
         end;
     false -> CONFIG
+end,
+case erlang:function_exported(rebar3, main, 1) of
+    true -> % rebar3
+        Config1;
+    false -> % rebar 2.x or older
+        %% Use git-based deps
+        %% profiles
+        [{deps, [{meck, ".*",{git, "https://github.com/eproxus/meck.git", {tag, "0.8.3"}}},
```

Comment:
I would rather prefer to have separate config for `rebar2` and invoke it through the make target with `rebar -C rebar2.config`